src/spec/mapping.spec.coffee

```diff
@@ -221,6 +221,38 @@ describe 'Mapping', ->
         done()
       .catch done
 
+    it 'should map empty search keywords', (done) ->
```

Comment:
Can we refactor this test and call mapSearchKeyword directly.

and we test this use case.